Support/Country Survey Details/PHL/LFS/Rmd/Administrative_Levels.Rmd

```diff
@@ -11,7 +11,7 @@ editor_options:
 knit: (function(input, ...) {
     rmarkdown::render(
       input,
-      output_dir = "/Volumes/OWLS/gld/Support/Country Survey Details/PHL/LFS",
```

Comment:
should resolve #84 but requires `here` and changes paradigm to project centered relative paths. Probably ok anyway